gradle.properties

```diff
@@ -1,27 +1,27 @@
 org.gradle.caching=true
 puppycrawlCheckstyleVersion=10.12.0
 group=org.ballerinalang
-version=1.8.0
+version=1.8.0-SNAPSHOT
 
-ballerinaLangVersion=2201.8.0
-stdlibIoVersion=1.6.0
-stdlibConstraintVersion=1.4.0
-stdlibOsVersion=1.8.0
-stdlibUuidVersion=1.7.0
-stdlibHttpVersion=2.10.0
-stdlibUrlVersion=2.4.0
-stdlibLogVersion=2.9.0
-stdlibTimeVersion=2.4.0
-stdlibMimeVersion=2.9.0
-stdlibAuthVersion=2.10.0
-stdlibCacheVersion=3.7.0
-stdlibCryptoVersion=2.5.0
-stdlibFileVersion=1.9.0
-stdlibTaskVersion=2.5.0
-stdlibJwtVersion=2.10.0
-stdlibOAuth2Version=2.10.0
+ballerinaLangVersion=2201.8.5
+stdlibIoVersion=1.6.0-20230911-134800-de06e28
```

Comment:
Bump these snapshot versions to the newest ones that 8.5 release uses

Reply:
Updated according to 2201.8.5 [dependencies](https://github.com/ballerina-platform/ballerina-distribution/blob/v2201.8.5/gradle.properties)